api/conf/urls/__init__.py

```diff
@@ -35,3 +39,6 @@
     path("healthcheck/", HealthCheck.as_view(), name="health"),
     path("v1/", include(versioned_paths)),
 ]
+
+if settings.ENVIRONMENT == "local":
+    urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
This is how `runserver` serves static files. Because we're using uvicorn for the local server now, we need to replicate that functionality of runserver.